spec/services/action_create_service_spec.rb

```diff
@@ -27,7 +27,8 @@
 
   context 'approve/deny operation' do
     it 'updates current stage' do
-      action = svc1.create(:operation => Action::APPROVE_OPERATION, :processed_by => 'man')
+      stage1.update_attributes(:state => Stage::NOTIFIED_STATE)
+      action = svc1.create('operation' => Action::APPROVE_OPERATION, 'processed_by' => 'man')
```

Comment:
@bzwei Can you explain why we seem to be switching calls from symbols to string, but adding new line that are using symbols?  This pattern repeats in this spec.  And the changes to `app/services/request_create_service.rb` are switching over to strings.  I'm not if there is a purpose or why it is not consistent.

Reply:
`update_attributes` as an AR method can take either symbol or string. 
`svc1.create` is not an AR method. It is normally called from the controller where it receives REST call with JSON body.